app/views/case_assignments/index.html.erb

```diff
@@ -1,29 +1,29 @@
-<h1><%= "#{t(".title")} #{@volunteer.display_name}" %></h1>
+<h1><%= "#{Editing} #{@volunteer.display_name}" %></h1>
```

Comment:
syntax issue

Reply:
@compwron GitHub states that this is merged. Am I correct in that I do not need to make a change to line 1?

Reply:
This is merged but the next step is to add a test to this PR https://github.com/rubyforgood/casa/pull/3552
Can you take on adding a test to this pr? :) https://github.com/rubyforgood/casa/pull/3552

Reply:
A couple of questions

- This issue is in the index view of the case_assignments_controller? I do not see an index method within that controller? Did I miss the method?

- There are no tests for the case_assignments_controller. Should I create those files and start writing tests for the view file?